task/views.py

```diff
@@ -139,44 +129,51 @@ def get(self, request, id=None):
         serializer = self.serializer_class(task, many=True)
         return Response(serializer.data)
     
-
+    
 class TaskCompleteView(APIView):
     serializer_class = TaskSerializer
     authentication_classes = [authentication.TokenAuthentication]
     
     def get(self, request, id=None):
-        task = Task.objects.filter(user__id=request.user.id)
-        title = request.data.get('title')
         if id:
-            task = task.filter(title=title)
-        serializer = self.serializer_class(task, many=True)
-        return Response(serializer.data)
+            task = Task.objects.get(id=id)
```

Comment:
I asked you to add validation. Validation should be there, right now anyone can check other's tasks by task id.